File: app/src/main/java/OnTrackBus/com/Usuarios.java
package OnTrackBus.com;

public class Usuarios {

    private String Nombre;
    private String Apellido;
    private String Correo;
    private String Contraseña;
    private String ContactoConfianza;
    private String RMF1=null,RMF2=null,RMF3=null;
    private String iduser;



    public Usuarios(){

    }

    public String getNombre() {
        return Nombre;
    }

    public void setNombre(String nombre) {
        Nombre = nombre;
    }

    public String getApellido() {
        return Apellido;
    }

    public void setApellido(String apellido) {
        Apellido = apellido;
    }

    public String getCorreo() {
        return Correo;
    }

    public void setCorreo(String correo) {
        Correo = correo;
    }

    public String getContraseña() {
        return Contraseña;
    }

    public void setContraseña(String contraseña) {
        Contraseña = contraseña;
    }


    public String getContactoConfianza() {
        return ContactoConfianza;
    }

    public void setContactoConfianza(String contactoConfianza) {
        ContactoConfianza = contactoConfianza;
    }


    public String getRMF1() {
        return RMF1;
    }

    public void setRMF1(String RMF1) {
        this.RMF1 = RMF1;
    }

    public String getRMF2() {
        return RMF2;
    }

    public void setRMF2(String RMF2) {
        this.RMF2 = RMF2;
    }

    public String getRMF3() {
        return RMF3;
    }

    public void setRMF3(String RMF3) {
        this.RMF3 = RMF3;
    }


    public String getIduser() {
        return iduser;
    }

    public void setIduser(String iduser) {
        this.iduser = iduser;
    }

}

File: app/src/main/java/com/android/ontrackbus/Models/Usuarios.kt
package com.android.ontrackbus.Models

class Usuarios {

    private var Nombre: String? = null
    private var Apellido: String? = null
    private var Correo: String? = null
    private var Contraseña: String? = null
    private var ContactoConfianza: String? = null
    private var RMF1: String? = null
    private var RMF2: String? = null
    private var RMF3: String? = null
    private var iduser: String? = null

    fun Usuarios() {}

    fun getNombre(): String? {
        return Nombre
    }

    fun setNombre(nombre: String?) {
        Nombre = nombre
    }

    fun getApellido(): String? {
        return Apellido
    }

    fun setApellido(apellido: String?) {
        Apellido = apellido
    }

    fun getCorreo(): String? {
        return Correo
    }

    fun setCorreo(correo: String?) {
        Correo = correo
    }

    fun getContraseña(): String? {
        return Contraseña
    }

    fun setContraseña(contraseña: String?) {
        Contraseña = contraseña
    }


    fun getContactoConfianza(): String? {
        return ContactoConfianza
    }

    fun setContactoConfianza(contactoConfianza: String?) {
        ContactoConfianza = contactoConfianza
    }


    fun getRMF1(): String? {
        return RMF1
    }

    fun setRMF1(RMF1: String?) {
        this.RMF1 = RMF1
    }

    fun getRMF2(): String? {
        return RMF2
    }

    fun setRMF2(RMF2: String) {
        this.RMF2 = RMF2
    }

    fun getRMF3(): String? {
        return RMF3
    }

    fun setRMF3(RMF3: String) {
        this.RMF3 = RMF3
    }


    fun getIduser(): String? {
        return iduser
    }

    fun setIduser(iduser: String?) {
        this.iduser = iduser
    }

}